transform/mattermost-analytics/models/staging/mattermost_nps/_mattermost_nps__models.yml

```diff
@@ -0,0 +1,84 @@
+version: 2
+
+models:
+  - name: stg_mattermost_nps__feedback
+    description: |
+      Reconstructed `tracks` table using custom properties expected to be in the events.
+
+    columns:
+      - name: event_id
+        description: The event's id.
+      - name: event_name
+        description: The name of the event.
+      - name: event_table
+        description: The name of the event table.
+      - name: user_id
+        description: The ID of the user that sent the event.
+      - name: server_id
+        description: The ID of the server the event originated from.
+      - name: received_at
+        description: Timestamp registered by Segment when the event was ingested (received).
+      - name: timestamp
+        description: Timestamp registered by Segment when the event occurred refactored to clients clock skew.
+      - name: user_created_at
+        description: Timestamp the user was created.
+      - name: license_id
+        description: The ID of the license.
+      - name: license_sku
+        description: The sku of the license.
+      - name: user_role
+        description: Role of the user who submitted NPS.   
+      - name: feedback
```

Comment:
It would be nice to test this is non-empty/null.

Reply:
I added a filter in the model to only fetch rows where `feedback` is not null. I don't think a test is required, since empty feedback rows can be excluded.

Reply:
There's no such check in `stg_mattermost_nps__nps_feedback` model. Tests help identify bugs and capture expectations.